lib/Doctrine/DBAL/Cache/ArrayStatement.php

```diff
@@ -19,7 +19,7 @@
 /**
  * @deprecated
  */
-class ArrayStatement implements IteratorAggregate, ResultStatement, Result
+class ArrayStatement implements IteratorAggregate, ResultStatement
```

Comment:
This is a breaking change. Please try using [Roave/BackwardCompatibilityCheck](https://github.com/Roave/BackwardCompatibilityCheck).

Reply:
Thanks for your quick return. It should be alright for BC break part now.